lblod/app-digitaal-loket
# [Toezicht module] Bundling migrations for besluit handhaven form adjustements

# Description
DL-5180

![Screenshot 2023-06-21 at 13-25-34 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/4dca83bb-8790-4f01-8f72-0e7f97b7b8aa)

This PR is about adjusting label and form for besluit handhaven.

- Adding a migration where we rename the label for "besluit handhaven" concept to "Besluit handhaven na ontvangst schorsingsbesluit".
- Adding semantic form migration where we include a new custom field "alert" where it shows the following alert message -> “Met dit formulier kan je een besluit van het bestuur handhaven, na ontvangst van het schorsingsbesluit van de provinciegouverneur of de toezichthoudende gemeente of provincie.“


# Type of change


# Related services

- [frontend-loket](https://github.com/lblod/frontend-loket) - bumping frontend

# How to test 

1. Checkout frontend-loket branch `chore/ember-submission-form-fields-2.11.0`
2. Start the stack, migrations should run.
3. Log as a worship-service 
4. Select Toezicht module and create a new submission with the dossier type of 'Besluit handhaven na ontvangst schorsingsbesluit'.
5. It should create the submission.

# What to check

- Should create a submission without errors.
- Existing submissions should have the new label.

# Links to other PR's

- https://github.com/lblod/manage-submission-form-tooling/pull/37
- https://github.com/lblod/frontend-loket/pull/321

# Notes
